packages/kbn-config-schema/index.ts

```diff
@@ -146,6 +146,23 @@ function recordOf<K extends string, V>(
   return new RecordOfType(keyType, valueType, options);
 }
 
+function oneOf<A, B, C, D, E, F, G, H, I, J, K, L>(
```

Comment:
Adding the ability to do oneOf 12 options 🙈

Reply:
OMG